.changeset/giant-dogs-crash.md

```diff
@@ -0,0 +1,6 @@
+---
+"@vue-storefront/next": major
+"@vue-storefront/nuxt": major
+---
+
+[ADDED] global state management
```

Comment:
Please add a bit more information about it and mark this release as a breaking change 

**References:**
- https://alokai.atlassian.net/wiki/spaces/KB/pages/335216697/Standardising+release+notes+and+migration+guides
- https://github.com/vuestorefront/vue-storefront/pull/7227/files#diff-7101df80574d26d88450f5116b0651b92b3db3307c397136eda549b939b2ad86